src/main/java/net/atos/zac/app/zaken/model/RESTZaakKoppelGegevens.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.app.zaken.model;
+
+import java.util.UUID;
+
+public class RESTZaakKoppelGegevens {
+    public UUID bronZaakUuid;
+    public RESTGerelateerdeZaak gerelateerdeZaak;
```

Comment:
Je geeft nu een RESTGerelateerdeZaak mee terwijl je de meeste info uit dat object niet nodig hebt om te kunnen koppelen. Gebruik dit object niet en geef in RESTZaakKoppelGegevens alleen de gegevens mee die je nodig hebt om te kunnen koppelen